frontend/src/components/EncryptedChat.js

```diff
@@ -43,6 +44,7 @@ class Chat extends Component {
     messages: [],
     value: '',
     connected: false,
+    connection: null,
```

Comment:
New but unused state variable.

Reply:
oh that's the problem, I don't how at some point it changes from `this.state.connection.send` to  `this.state.send`, I'll move it back, the idea of this object is to be the one who keep the connection with the server socket

Reply:
https://github.com/Reckless-Satoshi/robosats/pull/277/files#diff-cf6fbea242f7b24c53dea696717984a0fd76bde01795100ae85a8d44c26ac05eR78